client/components/Orders.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom';
+import { fetchOrder } from '../store'
+
+class Orders extends Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  async componentDidMount () {
+    try {
+      await this.props.fetchOrder()
+    }
+    catch(ex) {
+      next(ex)
+    }
+  }
+
+  render() {
+    const { order, auth } = this.props
+    return (
+      <div className='ordersinfo'>
+        <h2>Orders for { auth.username.slice(0, 1).toUpperCase() }{ auth.username.slice(1) }:</h2>
+        {
+          order.map(orders => {
```

Comment:
i hope the user's cart isnt being displayed here- because i think in our setup, carts and orders are the same database model